### PR TITLE
Full site for Prague 2021

### PR DESCRIPTION
**Pull request by mxsasha**
Includes enabling the full site and porting over Portland content that changed since making this site.

### CODE REVIEW

**Comment by mxsasha** [comment]
I think this also has the right pricing already, as we discussed it at the time.

**Comment by thatdocslady** [comment]
Looks great, a few comments here because I couldn't figure out how to make suggestions to pages that weren't changed:

- Let's remove the lightning talks sign-up form link for now (I'm sure it'll confuse some people...)
- Let's put the social event from 18:30-20:30 in the Schedule page
- Do we want to change the writing day times in the Writing Day page?
- Christy is going to update the WW FAQ section for Portland, we should pull it into this website when it's ready
- Team page shows Portland folks, let's clear all of them except the core team for now until we finalize the org team in a few weeks
- We should change the tickets_live variable to show May 2021

File: docs/conf/prague/2021/outing.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :template: {{year}}/generic.html
 
-{% if shortcode == 'prague' %}
+{% if shortcode == '{{ shortcode }}'%}
```
Comment (plaindocs) [inline docs/conf/prague/2021/outing.rst:3]: this looks like a potential mistake?

File: docs/conf/prague/2021/sponsors/prospectus.rst [hunk 2]
```diff
@@ -171,28 +171,26 @@ Please let us know if there is any information missing that would be useful for
 Sponsorship schedule
 --------------------
 
-* **SUNDAY**: The conference online platform is open. You are welcome to hang out at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also welcome to lead a Writing Day project if your documentation is open source.
+* **SUNDAY**: The conference online platform is open. You are welcome to hang out at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also encouraged to lead a Writing Day project if your documentation is open source.
 
-* **MONDAY**: The conference platform opens at 9:00, so we recommend arriving around this time to get the most interaction with attendees. The official start of the main conference talks is at 10:00, when sponsor booths will need to be staffed. The conference will run until around 18:00.
+* **MONDAY**: The conference platform opens at 8am, so we recommend arriving around this time to get the most interaction with attendees. The official start of the main conference talks is at 9am, when sponsor booths will need to be staffed. The conference will run until around 6pm.
```
Comment (plaindocs) [inline docs/conf/prague/2021/sponsors/prospectus.rst:176]: This feels early?

File: docs/conf/prague/2021/sponsors/prospectus.rst [hunk 3]
```diff
@@ -171,28 +171,26 @@ Please let us know if there is any information missing that would be useful for
...
 
 Sponsorship spaces
 ------------------
 
 A quick overview of the important spaces in the "venue":
 
-* The *main stage* is where the talks happen. This is also where Lightning talks will be given.
-* The *Sessions area* is where the Writing Day, Unconference and Social event will happen.
+* The *Main stage* is where the talks happen. This is also where Lightning talks will be given.
+* The *Sessions area* is where the Writing Day and Unconference will happen.
```
Comment (plaindocs) [inline docs/conf/prague/2021/sponsors/prospectus.rst:193]: & Q&A

**Review by plaindocs — APPROVED** [review]
:+1:  :+1:  :+1:

**Comment by mxsasha** [comment]
All comments addressed/fixed :)

**Comment by plaindocs** [comment]
@mxsasha I've merged in the CFP, hopefully without breaking anything.

File: docs/_data/prague-2020-schedule.yaml [hunk 4]
```diff
@@ -39,9 +39,9 @@ talks_day1:
     slug: bake-a-little-documentation-love-into-your-product-matt-reiner
   - time: '18:30'
     title: Day 1 wraps up
-  - time: '19:30'
```
Comment (plaindocs) [inline docs/_data/prague-2020-schedule.yaml]: This should be `docs/_data/prague-2021-schedule.yaml`

**Review by plaindocs — COMMENTED** [review]
Otherwise looks good.

File: docs/_data/prague-2020-schedule.yaml (hunk 4 above)
Comment (mxsasha) [reply docs/_data/prague-2020-schedule.yaml]: Oh I don't know why we touched this file at all? Think we can just undo it.

**Pull request merged**